src/components/leftCol/YourAccs.vue

```diff
@@ -4,9 +4,14 @@
             class="card custom-card is-flex-direction-column is-align-items-center is-justify-content-space-between custom-card-padding"
           >
             <h2 class="custom-card-title">vos comptes</h2>
+            <template v-if="moneyAccounts.length > 0">
             <Acc v-for="account in moneyAccounts" :bal="account.bal" :curr="account.curr">
               <template v-slot:name>{{ account.name.split(':')[0] }}</template>
             </Acc>
+            </template>
```

Comment:
This will be displayed while charging... and I feel this can be distressing. The best would be to have a placeholder while charging, and finish on that text if accounts.length is 0 and we know it.